app/controllers/chapters_controller.rb

```diff
@@ -55,7 +56,7 @@ def show
       @comments = @chapter.comments.reviewed
 
       @page_title = @work.unrevealed? ? ts("Mystery Work - Chapter %{position}", :position => @chapter.position.to_s) :
-        get_page_title(@work.fandoms.string,
+        get_page_title(@tag_groups["Fandom"][0].name,
           @work.anonymous? ? ts("Anonymous") : @work.pseuds.sort.collect(&:byline).join(', '),
```

Comment:
OK, having looked over the tests, the only thing I think needs coverage for this pullreq is the page title. There's page_title.feature but it's not actually checking for anything (including that the user preference or anonymity works at all :P) I suggest:

`Scenario: Page title should respect user preference

  Given I am logged in as "author"
    And I go to my preferences page
    And I fill in "Browser page title format" with "FANDOM - AUTHOR - TITLE"
    And I press "Update"
    And I post the work "New Story" with fandom "Stargate"
  When I view the work "New Story"
  Then the page title should include "Stargate - author - New Story"
  
Scenario: Page title should change when tags are edited
  Given I am logged in as "author"
    And I post the work "New Story" with fandom "Stargate"
  When I view the work "New Story"
  Then the page title should include "Stargate"
  When I edit the work "New Story"
    And I fill in "Fandoms" with "Harry Potter"
    And I post
  When I view the work "New Story"
  Then the page title should include "Harry Potter"
    And the page title should not include "Stargate"
  
`